### PR TITLE
Allow installs of Node Problem Detector during upgrades

### PR DESCRIPTION
**Pull request by joelsmith**

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/upgrade_components.yml [hunk 1]
```diff
@@ -25,3 +25,6 @@
 
 - import_playbook: ../../../openshift-monitor-availability/private/config.yml
   when: openshift_monitor_availability_install | default(false) | bool
+
+- import_playbook: ../../openshift-node-problem-detector/private/config.yml
```
Comment (vrutkovs) [inline playbooks/common/openshift-cluster/upgrades/upgrade_components.yml]: Needs additional `../`

Comment (joelsmith) [reply playbooks/common/openshift-cluster/upgrades/upgrade_components.yml]: Good catch. Thanks

**Comment by joelsmith** [comment]
@vrutkovs thanks for the review. I fixed the issue that you saw. Please review again. Thanks!

**Comment by vrutkovs** [comment]
I've noticed Origin version of problem detector doesn't have `v3.11` tag on dockerhub. 

@joelsmith are you planning to add release-based versions or use static (e.g. `v0.15`) version for this?

**Pull request merged**